kornia/core/module.py

```diff
@@ -273,3 +404,59 @@ def __call__(
         else:
             _output_image = super().__call__(*inputs, **kwargs)
         return _output_image
+
+
+class ImageSequential(Sequential, ImageModuleMixIn, ONNXExportMixin):
+    """Handles image-based operations as a sequential module.
+
+    This modules accepts multiple input and output data types, provides end-to-end
+    visualization, file saving features. Note that this module fits the classes that
+    return one image tensor only.
+
+    Note:
+        The additional add-on features increase the use of memories. To restore the
+        original behaviour, you may set `disable_features = True`.
+    """
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+        self._disable_features: bool = False
+
+    @property
+    def disable_features(self) -> bool:
+        return self._disable_features
+
+    @disable_features.setter
+    def disable_features(self, value: bool = True) -> None:
+        self._disable_features = value
+
+    def __call__(
```

Comment:
@shijianjian i think that here what we want to do is to support iobinding to inject directly torch or even other backers like Keras, jax, tf 
https://stackoverflow.com/questions/75553839/pre-allocating-dynamic-shaped-tensor-memory-for-onnx-runtime-inference